addons/sale_management/models/sale_order_option.py

```diff
@@ -91,6 +91,7 @@ def _compute_price_unit(self):
             option.price_unit = new_sol.price_unit
             # Avoid attaching the new line when called on template change
             new_sol.order_id = False
```

Comment:
This is not needed anymore if we drop the new record from the cache directly.

Reply:
Nice- I amended the comments directly preceding the lines as well to reflect the change (I guess we should keep them to explicitly justify why we call with `flush=False`? )

Should be good to go now.